packages/worker/src/tests/api/configs.ts

```diff
@@ -17,21 +22,44 @@ export class ConfigAPI extends TestAPI {
       .expect("Content-Type", /json/)
   }
 
-  getAIConfig = () => {
-    return this.request
+  getAIConfig = async () => {
+    const resp = await this.request
       .get(`/api/global/configs/ai`)
       .set(this.config.defaultHeaders())
       .expect(200)
       .expect("Content-Type", /json/)
+    return resp.body as AIConfig
   }
 
-  saveConfig = (data: any) => {
-    return this.request
+  saveConfig = async (
+    data: SaveConfigRequest,
+    expectations?: { status?: number; body?: Record<string, string | RegExp> }
+  ) => {
+    const { status = 200, body } = expectations || {}
+
+    const resp = await this.request
       .post(`/api/global/configs`)
       .send(data)
       .set(this.config.defaultHeaders())
-      .expect(200)
       .expect("Content-Type", /json/)
+
+    if (resp.status !== status) {
```

Comment:
Why this instead of `.expect(status)`?

Reply:
`.expect(status)` gives you no information about the content of the response, and makes debugging test failure really hard. It's why we moved away from it entirely in `packages/server` in favour of our own `Expectations`-based method.